File: cpp/0027_RemoveElement.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int res=0;
        int label=-1;
       
        for(int i=0;i<nums.size();i++)
        {
            while(i<nums.size()&&nums[i]==val)
            {
                 i++;
            }
            if(i>=nums.size())
                break;
             nums[res]=nums[i];
             label=nums[i];
             res++;
            
        }
        return res;
        
        
    }
};


class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
            nums.erase(remove(nums.begin(), nums.end(), val), nums.end());
        return nums.size();
    }
};

File: python/0027_RemoveElement.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        res=0
        for i in range(len(nums)):
            if(nums[i]==val):
                i+=1
            else:
                nums[res]=nums[i]
                res+=1

        return res


class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        while val in nums:
            nums.remove(val)
        return len(nums)


